src/coreclr/System.Private.CoreLib/src/System/Exception.CoreCLR.cs

```diff
@@ -181,7 +194,6 @@ internal void RestoreDispatchState(in DispatchState dispatchState)
         // DynamicMethodDescs alive for the lifetime of the exception. We do this because
```

Comment:
I assume that we use this array to also store loader allocator roots when collectible assemblies are referenced in the stacktrace. Is that correct? The names and comments that talk about DynamicMethods are misleading in that case.

Reply:
Yes, we do. I'll update the related comments.

Reply:
Actually, the renaming is very pervasive, let's make it as a separate change to not to cause more churn for the review.